.github/workflows/uberjar.yml

```diff
@@ -121,3 +124,34 @@ jobs:
     - name: Retag and push images if branch
       if: ${{ !(startsWith(github.ref_name,'master') || startsWith(github.ref_name,'main') || startsWith(github.ref_name,'backport')) && matrix.edition == 'ee' }}
       run: docker tag localhost:5000/metabase-dev:${{ steps.extract_branch.outputs.branch }}-ee metabase/metabase-dev:${{ steps.extract_branch.outputs.branch }} && docker push metabase/metabase-dev:${{ steps.extract_branch.outputs.branch }}
+
+  percy:
+    name: Percy visual test
+    if: github.ref_name == 'master' || github.ref_name == 'main' || startsWith(github.ref_name,'release-')
+    runs-on: ubuntu-20.04
```

Comment:
Didn't we determine that the regular ubuntu is not enough and that we should use custom buildjet runner?